mail-filter/bogofilter/bogofilter-1.2.5-r1.ebuild

```diff
@@ -0,0 +1,129 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools db-use flag-o-matic toolchain-funcs
+
+DESCRIPTION="Bayesian spam filter designed with fast algorithms, and tuned for speed"
+HOMEPAGE="http://bogofilter.sourceforge.net/"
+SRC_URI="mirror://sourceforge/${PN}/${P}.tar.xz"
+
+LICENSE="GPL-3+ CC-BY-ND-3.0"
```

Comment:
This still has the wrong licence? Anyway, I’d just git mv the bad ebuild and fix it in one commit?